[RHACS] ROX-18793: openshift monitoring (#63050)
openshift/openshift-docs: modules/enable-openshift-monitoring-central-helm.adoc

// Module included in the following assemblies:
//
// * configuration/monitor-acs.adoc
:_module-type: PROCEDURE
[id="enable-monitoring-central-helm_{context}"]
= Monitoring Central services using Helm charts

[role="_abstract"]

> You can monitor Central services, Central and Scanner, by changing the configuration options in the `central-services` Helm chart.

Reviewer: Today this option does not expose monitoring for scanner.

.Procedure
. Update the `values-public.yaml` configuration file with the following values:
+
[source,yaml]
----
monitoring.openshift.enabled: true
----
. Run the `helm upgrade` command and specify the configuration files using the `-f` option:
+
[source,terminal]
----
$ helm upgrade -n stackrox \
  stackrox-central-services rhacs/central-services \
  -f <path_to_values_public.yaml>
----
+
[NOTE]
====

> You can also specify configuration values using the `--set` or `--set-file` parameters. However, these options are not saved, and it requires you to manually specify all the options again whenever you make changes.

Reviewer 2: ```suggestion
You can also specify configuration values using the `--set` or `--set-file` parameters. However, these options are not saved, and you must manually specify all of the options again whenever you make changes.
```

Reviewer: Helm does save the options, and one can use them e.g. via `--reuse-values`. In any case, I don't think we should be too specific about how Helm works.

// Module included in the following assemblies:
//
// * configuration/monitor-acs.adoc
:_module-type: PROCEDURE
[id="enable-openshift-monitoring-central-helm_{context}"]
= Monitoring Central services by using Helm charts

[role="_abstract"]
You can monitor  Central and Central services, by changing the configuration options in the `central-services` Helm chart.

.Procedure
. Update the `values-public.yaml` configuration file with the following values:
+
[source,yaml]
----

> monitoring.openshift.enabled: true

Reviewer: This is already the default. Do we need this section?

Author: I think we should keep it. It's easier to reason how to disable it if you see how it's enabled.

Reviewer: Maybe we can condense it at least, as mentioned above I think we over-document now.
